src/networking.c

```diff
@@ -3042,6 +3042,10 @@ void clientCommand(client *c) {
 "      Kill connections authenticated by <username>.",
 "    * SKIPME (YES|NO)",
 "      Skip killing current connection (default: yes).",
+"    * ID <client-id>",
+"      Kill connections by client id.",
+"    * AGE <max-age>",
```

Comment:
```suggestion
"    * MAXAGE <max-age>",
```
Any reason to not be specific here?